app/containers/LoginLedgerNanoS/LoginLedgerNanoS.jsx

```diff
@@ -23,46 +28,84 @@ const { NOT_CONNECTED, OPEN_APP, CONNECTED } = LEDGER_CONNECTION_STAGES
 
 type LedgerConnectionStage = $Values<typeof LEDGER_CONNECTION_STAGES>
 
+type LedgerPublicKey = { account: number, key: string }
+
 type Props = {
   progress: string,
+  publicKey: LedgerPublicKey,
   login: Function,
   connect: Function,
-  publicKey: ?string,
   error: ?string
 }
 
+type SelectOption = {
+  label: string,
+  value: string
+}
```

Comment:
Better this be located in `StyledReactSelect.jsx` (used also in `Settings.jsx`)
And if needed to make more specific (e.g `PriceHistoryPanel` and `ContactsPanel`) we can do this:
```jsx
import type { SelectOption } from '../StyledReactSelect'

type Duration = '1m' | '1w' | '1d'

type DurationSelectOption = {
  ...SelectOption, // <-- copies type definition
  value: Duration // <-- overrides specific type
}
```

Wdyt? This can be a separate PR of course.

Reply:
Sure thats a great idea! However instead of exporting it from `StyledReactSelect` I will just add it to the flow-typed file where all of our global types are defined

Reply:
![awesome](https://user-images.githubusercontent.com/486954/50009976-f8562900-ffc0-11e8-9e8f-df6128154af0.jpeg)